docs/topics/maven.md

```diff
@@ -326,14 +336,12 @@ write a dependency on [`kotlin-bom`](https://mvnrepository.com/artifact/org.jetb
 </dependencyManagement>
 ```
 
-## Generating documentation
+## Generate documentation
 
-The standard Javadoc generation plugin (`maven-javadoc-plugin`) does not support Kotlin code.
-To generate documentation for Kotlin projects, use [Dokka](https://github.com/Kotlin/dokka);
-please refer to the [Dokka README](https://github.com/Kotlin/dokka/blob/master/README.md#using-the-maven-plugin)
-for configuration instructions. Dokka supports mixed-language projects and can generate output in multiple
-formats, including standard Javadoc.
+The standard Javadoc generation plugin (`maven-javadoc-plugin`) doesn't support Kotlin code. To generate documentation 
+for Kotlin projects, use [Dokka](https://github.com/Kotlin/dokka). For more information, see [Run Dokka – Maven](dokka-maven.md).
+Dokka supports mixed-language projects and can generate output in multiple formats, including standard Javadoc.
 
-## OSGi
+## Enable OSGi support
 
-For OSGi support see the [Kotlin OSGi page](kotlin-osgi.md).
+See [Kotlin and OSGi – Maven](kotlin-osgi.md#maven).
```

Comment:
Not sure that we use such notation for links.

Maybe:
Learn how to enable OSGi support in your Maven project?

Reply:
We never agreed on a convention 😄 I'll update with your suggestion 👍